spark/v3.2/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/SparkRowLevelOperationsTestBase.java

```diff
@@ -69,34 +70,34 @@ public SparkRowLevelOperationsTestBase(String catalogName, String implementation
       " format = {3}, vectorized = {4}, distributionMode = {5}")
   public static Object[][] parameters() {
     return new Object[][] {
-        { "testhive", SparkCatalog.class.getName(),
-            ImmutableMap.of(
-                "type", "hive",
-                "default-namespace", "default"
-            ),
-            "orc",
-            true,
-            WRITE_DISTRIBUTION_MODE_NONE
+        {"testhive", SparkCatalog.class.getName(),
+         ImmutableMap.of(
+             "type", "hive",
+             "default-namespace", "default"
+         ),
+         "orc",
+         true,
+         WRITE_DISTRIBUTION_MODE_NONE
```

Comment:
Please remove all unnecessary formatting changes so that we can see just what changed. Thanks!

Reply:
Done. In addition, I roll back the changes that were migrated to the new test  class from the last commit. Wait until the new test class is completed.